### PR TITLE
ref(codecov): Add line number anchor to codecov URL

### PR DESCRIPTION
**Pull request by jangjodi**
WOR-2556

### CODE REVIEW

File: static/app/components/events/interfaces/frame/stacktraceLink.tsx [hunk 1]
```diff
@@ -291,7 +291,7 @@ export function StacktraceLink({frame, event, line}: StacktraceLinkProps) {
         </OpenInLink>
         {shouldshowCodecovFeatures(organization, match) && (
           <CodecovLink
-            codecovUrl={match.codecov?.coverageUrl}
+            codecovUrl={`${match.codecov?.coverageUrl}#L${frame.lineNo}`}
```
Comment (snigdhas) [inline static/app/components/events/interfaces/frame/stacktraceLink.tsx:294]: Looks great! let's update the test for this

**Comment by evanpurkhiser** [comment]
Why's it a placeholder?

File: static/app/components/events/interfaces/frame/stacktraceLink.tsx (hunk 1 above)
Comment (jangjodi) [reply static/app/components/events/interfaces/frame/stacktraceLink.tsx:294]: Good call, thank you!

**Comment by snigdhas** [comment]
> Why's it a placeholder?

Not really a placeholder, the ticket should've been reworded

**Review by snigdhas — APPROVED** [review]

**Pull request merged**